Application/EdFi.Ods.Api/Controllers/VersionController.cs

```diff
@@ -126,7 +129,7 @@ public IActionResult Get()
 
                 if (_apiSettings.IsFeatureEnabled(ApiFeature.XsdMetadata.GetConfigKeyName()))
                 {
-                    urlsByName["xsdMetadata"] = Request.RootUrl(useReverseProxyHeaders) + "/metadata/" +
+                    urlsByName["xsdMetadata"] = rootUrl + "/metadata/" +
```

Comment:
There are 3 places that need to be updated to reference ``rootUrl`` as well, see ``changeQueries``, ``composites`` and ``identity`` sections a few lines below.

Reply:
done, thanks for pointing those out - i missed them in the cherry pick, as those lines don't exist in 5.3.